.babelrc

```diff
@@ -1,13 +1,22 @@
 {
   "presets": [
-    ["@babel/env", {
-      "targets": {"node": "10"}
-    }],
+    [
+      "@babel/env", {
+        "targets": {"node": "14"},
+        // Leave import/export statements unchanged in the babel transpiling output.
+        "modules": false
+      }
+    ],
     "@babel/flow"
   ],
   "plugins": [
     "@babel/plugin-proposal-class-properties",
```

Comment:
I see this is included in ES2022 and we're using `preset-env` so could this plugin be removed?

Reply:
yep, it looks like we can remove both this one and the one that follows it `@babel/plugin-transform-runtime`, this last one doesn't actually seem to make any actual difference in the resulting transpiled modules, I double-checked that locally by looking to the diffs).

Removing `@babel/plugin-proposal-class-properties` instead gets rid of a imported module to polyfill that using defineProperty:

```
--- lib.with-plugin-class-properties/util/temp-dir.js        2022-06-01 12:15:39.783001797 +0200
+++ lib/util/temp-dir.js    2022-06-01 12:14:38.777841288 +0200
@@ -1,4 +1,3 @@
-import _defineProperty from "@babel/runtime/helpers/defineProperty";
 import { promisify } from 'util';
 import tmp from 'tmp';
 import { createLogger } from './logger.js';
@@ -46,10 +45,6 @@

 export class TempDir {
   constructor() {
-    _defineProperty(this, "_path", void 0);
-
-    _defineProperty(this, "_removeTempDir", void 0);
-
     this._path = undefined;
     this._removeTempDir = undefined;
   }
```

The unit tests did pass just fine without those _defineProperty calls on both node 14 and node 16, and the unit tests are using the transpiled modules and so polyfilling that was not really necessary anymore.

Based on the compatibility table on MDN, that JS syntax should be supported in nodejs starting from nodejs 12.

I'll update this PR accordingly.